accel-pppd/ctrl/pppoe/cli.c

```diff
@@ -53,6 +54,11 @@ static void intf_help(char * const *fields, int fields_cnt, void *client)
 			 "pppoe interface show"
 			 " - show interfaces on which pppoe server"
 			 " started\r\n");
+	if (show & 8)
+		cli_send(client,
+			 "pppoe interface monitored"
+			 " - show list of monitored interfaces, pppoe service"
+			 " would be launched automatically when the link appears\r\n");
```

Comment:
```suggestion
			 " - show the list of monitored interfaces where PPPoE service"
			 " would be launched automatically when the link appears\r\n");
```

That help string is rather wordy but maybe let's go with that for now.